functorch/experimental/_map.py

```diff
@@ -74,9 +74,15 @@ def create_fw_bw_graph(f, num_mapped_args, *args):
 
             def from_fun(t):
                 if isinstance(t, torch.Tensor):
-                    return torch.empty_strided(
-                        t.size(), t.stride(), requires_grad=t.requires_grad
-                    )
+                    if t.dtype != torch.bool:
```

Comment:
What's going on here?

Reply:
boolean tensor created by empty_strided is not properly intialized so it causes error "load of value 190, which is not a valid value for type 'bool'" on clang asan environment on CI.